apps/src/studio/studio.js

```diff
@@ -1511,7 +1511,9 @@ function updateItems() {
     }
   }
   Studio.sprite.forEach(sprite => {
-    sprite.update();
+    if (sprite.visible) {
+      sprite.update();
```

Comment:
I'm slightly worried this could have unintentional side effects; what was the motivation for this change?